### ISSUE TITLE
Nest Cameras (new battery) don't work on macOS

### ISSUE THREAD

**Issue by Hamish1066** [home-assistant/iOS#2072]
Model Name: MacBook Pro (Retina, 15-inch, Late 2013)
macOS Version: 11.6.3 (20G415)
App Version: 2022.2
Safari Version: 15.3

Home Assistant Core Version
core-2022.2.2

**Describe the bug**
The new Nest (battery) cameras don't display on the macOS app - a blank black screen is shown. The older Nest Doorbell camera works fine on the macOS app. All Nest cameras work properly on Safari. All cameras work properly in iOS.

**To Reproduce**
View Nest cameras (battery and Nest doorbell) on macOS app. 

**Expected behavior**
All cameras should display properly on the macOS app as they do in Safari and iOS.

**Comment by allenporter** [comment]
The nest webrtc cameras have a special player support in frontend, so I'm wondering if they need special love to get working in the Mac app.

**Comment by zacwest** [comment]
I can reproduce the issue. Is there a User-Agent or something check in the frontend for the cameras to decide how to support it?

**Comment by allenporter** [comment]
I am not aware of a user agent check happening.  Though it does probe for some capabilities.

There is a check that `RTCPeerConnection` is a valid object here: https://github.com/home-assistant/frontend/blob/e95065ed08cafbc5a950ab79bf03f946766f7f99/src/components/ha-camera-stream.ts#L123

Then there is also check for the `frontend_stream_type` which comes from core however: https://github.com/home-assistant/frontend/blob/dev/src/components/ha-camera-stream.ts#L100

**Comment by bgoncal** [comment]
WebRTC is not supported in mac catalyst (which is how we ported the iOS app to mac), so this won't work unfortunately

I will be closing the issue since is quite old, the latest core release provide a new native player with some fallback options, perhaps one of those can help.
Let me know if I should reopen this one.

**Comment by allenporter** [comment]
I agree on the main part of the comment here about missing mac support -- but to clarify, there is no fallback. This is a native WebRTC camera and there is no HLS stream to fall back to if I understand your point.

**Comment by bgoncal** [comment]
Can you check if opening in Safari you get a fallback? This is controlled by the frontend, so on the apps side there is not much that can be done

**Comment by allenporter** [comment]
A fallback to what?

**Comment by allenporter** [comment]
Keep in mind this issue was reported in Feb 2022 when Nest native WebRTC cams were the only webrtc cams in home assistant. I agree with your assessment of the issue that mac doesn't have this capability. 

I'm just clarifying there is no fallback here since these cameras can't do HLS. Other cameras using go2rtc and HLS/RTSP should be able to support fallback.

**Comment by bgoncal** [comment]
Got it, thanks for clarifying

**Comment by dangrin0** [comment]
I can confirm that the Nest camera live view does work in Safari, rendered within a ha-web-rtc-player tag

**Comment by bgoncal** [comment]
@dangrin0 correct, because Safari supports WebRTC, unfortunately the WKWebView used in a Mac Catalyst app doesn't.
